app/components/Dropdown/MenuStyled.js

```diff
@@ -2,7 +2,27 @@ import styled from 'styled-components';
 
 import { BG_LIGHT, BORDER_RADIUS_L } from 'style-constants';
 
+const topProp = {
```

Comment:
this option will not work, because when adding new dropdowns, you need to add what is new, the dropdown should be universal and its content should be shown automatically in the right place